server/services/group.js

```diff
@@ -19,6 +21,17 @@ function deleteGroup(groupName) {
   return Group.delete(groupName);
 }
 
+function canRemoveAdmin(groupName) {
```

Comment:
I think I'd rather just have it not return anything and just do the check and throw if there is a sole admin